sdk/core/core-lro/test/utils/testOperation.ts

```diff
@@ -22,6 +21,7 @@ export interface TestOperationState extends PollOperationState<string> {
 
 export interface TestOperation extends PollOperation<TestOperationState, string> {}
 
+/** @this Mocha.Context */
```

Comment:
Isnt "this" referring to `TestOperation` here?

Reply:
You're right, I did a simple find and replace and did not pay attention to the rare cases. Thanks!